scripts/lokalise/string_resources.rb

```diff
@@ -35,6 +35,12 @@ def fetch
                 # Get the main element information
                 key_name = element.attributes['name']
                 value = element.text
+
+                if value.nil?
+                    puts "Warning: The string for key '#{key_name}' is nil and will be skipped."
+                    next  # Skip this iteration if value is nil
+                end
```

Comment:
Why was this added?

Reply:
This helps seeing if a key was skipped due to wrong formatting (that was the case in FC), and continues pushing the keys that are still valid.